src/server/game/Conditions/ConditionMgr.cpp

```diff
@@ -1259,32 +1259,33 @@ bool ConditionMgr::isSourceTypeValid(Condition* cond)
 {
     if (cond->SourceType == CONDITION_SOURCE_TYPE_NONE || cond->SourceType >= CONDITION_SOURCE_TYPE_MAX)
     {
-        sLog->outErrorDb("Invalid ConditionSourceType %u in `condition` table, ignoring.", uint32(cond->SourceType));
+        sLog->outErrorDb("Invalid ConditionSourceType %u in `conditions` table, ignoring.", uint32(cond->SourceType));
         return false;
     }
 
     switch (cond->SourceType)
     {
-        case CONDITION_SOURCE_TYPE_TERRAIN_SWAP:
-        case CONDITION_SOURCE_TYPE_PHASE:
-        case CONDITION_SOURCE_TYPE_GRAVEYARD:
+        case CONDITION_SOURCE_TYPE_UNUSED_20:
+        case CONDITION_SOURCE_TYPE_UNUSED_25:
+        case CONDITION_SOURCE_TYPE_UNUSED_26:
+        case CONDITION_SOURCE_TYPE_UNUSED_27:
         {
-            sLog->outErrorDb("ConditionSourceType %u in `condition` table is not supported on 3.3.5a, ignoring.", uint32(cond->SourceType));
+            sLog->outErrorDb("ConditionSourceType = (%u) in `conditions` is not in use. SourceEntry = (%u), skipped", uint32(cond->SourceType), cond->SourceEntry);
```

Comment:
is not in use?? that doesn't seem to convey the right information to the user

user will think "ah i need to change this setting to make it 'in use'" or smthg like that

what do you mean by "is not in use" ?? i think you mean "is not implemented yet" which is much clearer

Reply:
Well not in use is something me and Shin made up before.
But not implemented also feels wrong, I'd settle with "is a placeholder" if that sounds ok?